### ISSUE TITLE
Fails installing

### ISSUE THREAD

**Issue by SpaceboyRoss01** [BlueLightOS/osjs-bluelight-installer-application#1]
1. Cannot create `/home/osjs`
2. `umount /mnt` fails because target is busy

**Comment by SpaceboyRoss01** [comment]
3. Linux kernel is not found
```
==> Building image from preset: /etc/mkinitcpio.d/linux.preset: 'default'
  -> -k /boot/vmlinuz-linux -c /etc/mkinitcpio.conf -g /boot/initramfs-linux.img
==> ERROR: specified kernel image does not exist: `/boot/vmlinuz-linux'
==> Building image from preset: /etc/mkinitcpio.d/linux.preset: 'fallback'
  -> -k /boot/vmlinuz-linux -c /etc/mkinitcpio.conf -g /boot/initramfs-linux-fallback.img -S autodetect
==> ERROR: specified kernel image does not exist: `/boot/vmlinuz-linux'
[root@osjs /]# <rub-install --target=i386-pc --recheck /dev/sda; fi
```

**Comment by SpaceboyRoss01** [comment]
4. EFI directory is not found
```
Installing for x86_64-efi platform.
grub-install: error: cannot find EFI directory
```

**Comment by SpaceboyRoss01** [comment]
Forgot to close this, lmao06.01.2019, 22:42, "SpaceboyRoss01" <notifications@github.com>:Closed #1.
